.github/workflows/deploy-cpp.yml

```diff
@@ -37,6 +37,6 @@ jobs:
     - name: Generate artifact attestation for sdist and wheel
       uses: actions/attest-build-provenance@173725a1209d09b31f9d30a3890cf2757ebbff0d # v1.1.2
       with:
-        subject-path: "dist/awkward-cpp-*"
+        subject-path: "dist/awkward*cpp-*"
```

Comment:
c.f. https://github.com/scikit-hep/awkward/pull/3126#issuecomment-2136079341

Reply:
This is fine. Nothing else will be named in such a way that starts with `awkward` and ends with `cpp`.

<p>Maybe <tt>awkward-robocpp</tt>? 
<img src="https://github.com/scikit-hep/awkward/assets/1852447/7cd6c01d-6d27-4d1d-9380-1a0b384dc4bd" width="24"></p>